### ISSUE TITLE
Error on running the docker container

### ISSUE THREAD

**Issue by HeartedApp** [satishbabariya/theia-swift-docker#1]
I’m getting this error on running the docker container, any help is appreciated.

```
docker run -it -p 3000:3000 -v "/Users/justin/Workspace:/home/project:cached" satishbabariya/theia-swift:next --name theia
/home/theia/node_modules/@theia/core/lib/common/uri.js:33
vscode_uri_1.setUriThrowOnMissingScheme(false);
             ^

TypeError: vscode_uri_1.setUriThrowOnMissingScheme is not a function
    at Object.<anonymous> (/home/theia/node_modules/@theia/core/lib/common/uri.js:33:14)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:690:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/home/theia/node_modules/@theia/core/lib/common/selection.js:18:13)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
```

**Comment by HeartedApp** [comment]
FWIW the reason I’m trying your repo out is I get this error from the main theiaide/theia-swift:next docker image:

https://forums.swift.org/t/sourcekit-lsp-not-finding-modules-vscode/25799

**Comment by satishbabariya** [comment]
@HeartedApp, can you try this one `docker run -it -p 3000:3000 theiaide/theia-swift:next`, because this repo mostly used for further development for theia swift extension so it can be unstable sometimes.

**Comment by HeartedApp** [comment]
@satishbabariya Yeah I tried theiaide/theia-swift:next before going to this repo, the docker container works but I'm unable to import any modules from my swift project due to this issue: https://forums.swift.org/t/sourcekit-lsp-not-finding-modules-vscode/25799.

`error: unknown argument: '-package-description-version'`

Which they say is a mismatch between swiftpm 5.1 with swiftc 5.0.

That was a few weeks ago so I'll try that docker container out again and I see you've made some commits here as well so I can try this repo out again and let you know.

**Comment by HeartedApp** [comment]
@satishbabariya I just tried the theiaide/theia-swift:next docker image again but still run into the same problem.  Basically no local models can be imported

```[sourcekitd]
no such module 'EmailAdapters'(2, 8)
```
When I looked into this before it lead to the link above.  I also tried to use the satishbabariya/theia-swift:next docker image but ran into the same error as my first post in this issue.

Is there a specific tag on either of these docker images that is known to work with swift packages, if so that would get me by.  Thanks.
